src/models/FGAuxiliary.cpp

```diff
@@ -186,11 +192,6 @@ bool FGAuxiliary::Run(bool Holding)
 
   // Position tracking in local frame with local frame origin at lat, lon of initial condition
   // and at 0 altitude relative to the reference ellipsoid. Position is NEU (North, East, UP) in feet.
-  if (!NEUFromStartInitialized) {
-    NEUStartLocation = FDMExec->GetIC()->GetPosition();
-    NEUStartLocation.SetPositionGeodetic(NEUStartLocation.GetLongitude(), NEUStartLocation.GetGeodLatitudeRad(), 0.0);
-    NEUFromStartInitialized = true;
-  }
   vNEUFromStart = NEUStartLocation.LocationToLocal(in.vLocation);
   vNEUFromStart(3) *= -1.0;  // Flip sign for Up, so + for altitude above reference ellipsoid
```

Comment:
Don't we want this value to be computed on demand ? i.e. if and only of the property `position/from-start-neu-*ft` is read ?

That would mean the vector `vNEUFromStart` would be updated by the method `FGAuxiliary::GetNEUPositionFromStart`.